src/civetweb.c

```diff
@@ -192,7 +180,25 @@ clock_gettime(int clk_id, struct timespec *t)
 	}
 	return -1; /* EINVAL - Clock ID is unknown */
 }
+
+/* if clock_gettime is declared, then __CLOCK_AVAILABILITY will be defined */
```

Comment:
Both - since __CLOCK_AVAILABILITY is undef'd  at the end of a block that this change tries to detect, the patch will effectively always fall back to line 199. You can see how it looks like in xcode for example here: https://gist.github.com/kode54/14a22fa6626cc7e867896505de0280b5